src/Tribe/Tickets_Handler.php

```diff
@@ -1063,15 +1052,15 @@ public function save_tickets_order( $post_id ) {
 
 		// If our data is missing or we're already in the middle of saving, bail
 		if (
-			empty( $_POST[ 'tribe_tickets_order' ] ) ||
-			! empty( $this->saving_order ) ||
-			! ( isset( $_POST[ 'tribe-tickets-post-settings' ] ) && wp_verify_nonce( $_POST[ 'tribe-tickets-post-settings' ], 'tribe-tickets-meta-box' ) )
+			empty( $_POST[ 'tribe_tickets_order' ] )
+			|| ! (
+				isset( $_POST[ 'tribe-tickets-post-settings' ] )
```

Comment:
As per our [multi-line conditional statement standards](http://moderntribe.github.io/products-engineering/php/#php-control-multi), the logical operator should be on the same line as the next conditional statement.